src/core/Akka.Remote.Tests/PhiAccrualModelBasedSpecs.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Akka.Util;
+using FluentAssertions;
+using FsCheck;
+using FsCheck.Experimental;
+using FsCheck.Xunit;
+using Microsoft.FSharp.Core;
+using Xunit;
+
+namespace Akka.Remote.Tests
+{
+    public class PhiAccrualModelBasedSpecs
```

Comment:
Added model-based test here just to make sure all of the math was being done correctly inside the `PhiAccrualFailureDetector`.